AtDigitization/AtVectorResponse.cxx

```diff
@@ -0,0 +1,49 @@
+#include "AtVectorResponse.h"
+
+#include "AtDigiPar.h"
+
+#include <FairRunAna.h>
+#include <FairRuntimeDb.h>
+#include <FairTask.h>
```

Comment:
Remove

Reply:
IDE disagrees with that action.

Reply:
I disagree with the IDE, it compiles with FairTask removed.